mangaki/mangaki/migrations/0099_work_visible.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.27 on 2022-08-05 13:57
```

Comment:
What is the relation with this PR and this new field?

Reply:
Je sais faut juste que je l'ajoute aux bullet points :) Je ne voulais pas faire une PR d'une ligne.